tests/foreman/cli/test_satellitesync.py

```diff
@@ -581,19 +582,26 @@ def test_positive_export_import_cv_end_to_end(
         exported_packages = Package.list({'content-view-version-id': export_cvv_id})
         assert len(exported_packages)
         # Verify export directory is empty
-        assert validate_filepath(default_sat, module_org) == ''
+        assert validate_filepath(target_sat) == ''
         # Export cv
-        export = ContentExport.completeVersion(
+        path = ContentExport.completeVersion(
             {'id': export_cvv_id, 'organization-id': module_org.id}
         )
-        import_path = move_pulp_archive(default_sat, module_org, export['message'])
-
+        # grab export dir and check all exported files are there
```

Comment:
Significant change to test content that maybe should not be backported to 6.10.z, @rmynar please review.

Reply:
Please keep this function call...
Must have been some wrongly resolved merge conflict
```suggestion
        import_path = move_pulp_archive(default_sat, module_org, export['message'])
```